orderer/mocks/common/blockcutter/blockcutter.go

```diff
@@ -103,3 +103,39 @@ func (mbc *Receiver) CurBatch() []*cb.Envelope {
 	defer mbc.mutex.Unlock()
 	return mbc.curBatch
 }
+
+// SetIsolatedTx is used to change IsolatedTx field in order to avoid data race in the tests
+// If the field is changed directly from the test, a data race is created between the thread using the block cutter
+// and the test thread setting the field's value.
```

Comment:
You don't need to explain why we did it in the comment, but in the issue / PR comments / commit message.

Here, just
// SetIsolatedTx is used to change the IsolatedTx field in a thread safe manner.

Moreover, since those fields should never be accessed from the outside, change their scope, i.e. `IsolatedTx  -> isolatedTx`
 
Same for all other fields.

Reply:
done!